react/SectionDivider/styles.styl

```diff
@@ -0,0 +1,26 @@
+.divider
+  align-items center
+  display flex
+
+  &::after,
+  &::before {
+    content ''
+    height 1px
+    background-color var(--silver)
```

Comment:
`silver` ne va pas changer de couleur selon les thèmes, on est bon quand même, même un thème dark ?

Reply:
Yes le theme dark du browser ne change pas cette var. Le mieux serait d'avoir une vraie variable pour cette couleur mais je n'ai pas trouvé

Reply:
Si tu trouves pas, @joel-costa se fera un plaisir de t'aiguiller ou de la créer ;)

Reply:
In progress

Reply:
@JF-Cozy j'ai mis à jour, si ça te va je merge